pkg/controller/bootstrap/tokencleaner.go

```diff
@@ -161,9 +161,7 @@ func (tc *TokenCleaner) processNextWorkItem() bool {
 
 func (tc *TokenCleaner) syncFunc(key string) error {
 	startTime := time.Now()
-	defer func() {
-		klog.V(4).Infof("Finished syncing secret %q (%v)", key, time.Since(startTime))
-	}()
+	defer klog.V(4).Infof("Finished syncing secret %q (%v)", key, time.Since(startTime))
```

Comment:
This breaks the timing code. The function is needed to defer the call to `time.Since(startTime)` as well, which otherwise is called immediately.  Have a look at https://play.golang.org/p/qqXkfJKEEuI

Reply:
Sorry, I didn't pay attention to the passing time of deferred function. 😵